README.md

```diff
@@ -205,6 +205,7 @@ hazelcast:
 Cloud SSL configuration:
 ```
 ssl:
+    servername: "hazelcast.cloud"
```

Comment:
How about setting this by default if the cloud is enabled, so the user doesn't have to configure it?